pom.xml

```diff
@@ -355,7 +355,7 @@
     <mockito.version>2.27.0</mockito.version>
     <hamcrest.version>1.3</hamcrest.version>
     <commons-cli.version>1.2</commons-cli.version>
-    <netty.version>4.1.73.Final</netty.version>
+    <netty.version>4.1.75.Final</netty.version>
```

Comment:
Please raise PR first on master,
newer version is already available, lets upgrade to 4.1.76.Final

Reply:
+1